index.html

```diff
@@ -5502,7 +5504,101 @@ <h2>Base Context</h2>
   }
 }
 </pre>
+      </section>
+
+      <section class="informative">
+        <h3>Differences between Contexts, Types, and CredentialSchemas</h3>
 
+        <p>
+The verifiable credential and verifiable presentation data models leverage a
+variety of underlying technologies including [JSON-LD] and [JSON-SCHEMA-2018].
+This section will provide a comparison of the <code>@context</code>,
+<code>type</code>, and <code>credentialSchema</code> properties, and cover some
+of the more specific use cases where it is possible to use these features of the
+data model.
+        </p>
+        
+        <p>
+The primary purpose of the <code>type</code> property is to uniquely
+identify the type of the <code>verifiableCredential</code> in which it
```

Comment:
`type` is used to specify the type of /any/ object, not just the VC. The language should be clarified to say that.

Reply:
> In the context of this informative text isn't the top level type only used to uniquely identify the VC. Nested types identify the types of the nested objects.

Yes, that's true, but the current text doesn't make that differentiation. It just says "The primary purpose of the <code>type</code> property is to uniquely ￼identify the type of the <code>verifiableCredential</code>" ... which is, of course, not true for nested objects.

Reply:
I've removed "primary purpose of the" and added additional text to the bottom of the paragraph to talk more about `type` usage across the VC for any objects. I believe this should address your concern @msporny